app/src/main/java/com/droidhats/campuscompass/views/ExploreCategoryFragment.kt

```diff
@@ -0,0 +1,107 @@
+package com.droidhats.campuscompass.views
+
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.droidhats.campuscompass.R
+import com.droidhats.campuscompass.adapters.ExplorePlaceAdapter
+import com.droidhats.campuscompass.models.Explore_Place
+import com.droidhats.campuscompass.viewmodels.ExplorePlaceViewModel
+
+class ExploreCategoryFragment: Fragment() ,AdapterView.OnItemSelectedListener {
+
+    private lateinit var root : View
+    private lateinit var viewModel: ExplorePlaceViewModel
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var places: ArrayList<Explore_Place>
+    private var columnCount = 1
+
+    companion object {
+        var onExplorePlaceClickListener: OnExplorePlaceClickListener? = null
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        viewModel = ViewModelProviders.of(this).get(ExplorePlaceViewModel::class.java)
+        super.onCreate(savedInstanceState)
+
```

Comment:
I would suggest we remove the extra line

Reply:
Fixed